examples/lights/lights.c

```diff
@@ -0,0 +1,73 @@
+/**
+ * Copyright (C) 2009 Ubixum, Inc. 
+ * Copyright (C) 2016 Matthew Iselin <miselin@google.com>
+ * Copyright (C) 2017 Kyle Robbertze <krobbertze@gmail.com>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301  USA
+ **/
+
```

Comment:
Formatting of this file is not consistent.

Please check things like tabs / spaces / alignment of braces / etc. You might want to look at https://github.com/GStreamer/common/blob/master/gst-indent or the kernel format-patch stuff to check this.